src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothOppImpl.cs

```diff
@@ -39,7 +39,7 @@ internal class BluetoothOppServerImpl
             }
             remove
             {
-                //nothing to be done
+                _ConnectionRequested -= value;
```

Comment:
Why don't you use _ConnectionRequested instead directly use ConnectionRequested?

``` c#
//Remove _ConnectionRequested;
internal event EventHandler<ConnectionRequestedEventArgs> ConnectionRequested;

...
ConnectionRequested?.Invoke(this, new ConnectionRequestedEventArgs(devAddress));
```

Reply:
Ok, did you want just to fix a TCT issue?
But, Many hidden issue exist in BluetoothOpp class

Reply:
Right, Need to fix C# BT Manual TCT related with C# BT OPP API
I will memorize your comment and plan to handle it. (directly use ConnectionRequested)